README.md

```diff
@@ -38,9 +38,19 @@ Deprecation Notice: The Java interfaces are removed from the oneDAL library.
 
 ## Installation
 
-Check [System Requirements](https://oneapi-src.github.io/oneDAL/system-requirements.html) before installing oneDAL.
+Check the [System Requirements](https://oneapi-src.github.io/oneDAL/system-requirements.html) before installing to ensure compatibility with your system.
+
+There are several options available for installing oneDAL:
+
+- **Binary Distribution**: You can download pre-built binary packages from the following sources:
+    - Intel® oneAPI:
+        - Download as Part of the [Intel® oneAPI Base Toolkit](https://www.intel.com/content/www/us/en/developer/tools/oneapi/onedal.html#gs.8xrue2)
+        - Download as the Stand-Alone [Intel® oneAPI Data Analytics Library](https://www.intel.com/content/www/us/en/developer/tools/oneapi/onedal.html#gs.8xrue2)
+    - [Anaconda](https://anaconda.org/intel/daal4py) (as daal4py)
+    - [NuGet](https://www.nuget.org/packages/Microsoft.ML.OneDal)
```

Comment:
@napetrov, do we need these options here ?

Reply:
yes, but let's put link to base nuget package for onedal, not one that microsfot creating currently 
```suggestion
    - [Anaconda](https://anaconda.org/intel/daal4py) (as daal4py)
    - [NuGet](https://www.nuget.org/packages/inteldal.devel.linux-x64)
```